openapi/products/listingBuilder.yaml

```diff
@@ -0,0 +1,178 @@
+openapi: 3.0.0
+info:
+  title: Reputation REST APIs
+  version: '1.0'
+  description: ''
+servers:
+  - url: 'https://prod.apigateway.co/products/listingBuilder/listListings'
+    description: Production
+  - url: 'https://demo.apigateway.co/products/listingBuilder/listListings'
+    description: Demo
+  - url: '{local}/products/listingBuilder/listListings'
+    description: Local
+  - url: 'http://localhost:11001/products/listingBuilder/listListings'
+    description: Localhost
+paths:
+  /listListings:
```

Comment:
The path and model name should be `listings`.

There are example paths for each operation [here](https://vendasta.jira.com/wiki/spaces/API/pages/819626067/Endpoint+Design+Considerations#Operations)